src/tasks/tasks.py

```diff
@@ -74,13 +73,22 @@ def process_single_pair_task_split(split, indexers, is_pair=True, classification
     Returns:
         - instances (Iterable[Instance]): an iterable of AllenNLP Instances with fields
     '''
+    # check here if using bert to avoid passing model info to tasks
+    bert_pair = max([True if "bert" in idx_name else False for idx_name in indexers])
```

Comment:
I'm not sure what this is doing - consider using `any(...)` instead if checking for existence?